src/main.rs

```diff
@@ -1,35 +1,25 @@
+mod request_methods;
+mod args;
+
 use clap::Parser;
 use std::fs;
 use std::io::prelude::*;
 use std::net::TcpListener;
 use std::net::TcpStream;
+use std::str::from_utf8;
+use request_methods::RequestMethods;
+use args::Args;
 
-const DEFAULT_IP: &str = "127.0.0.1";
+const IP_ADDRESS: &str = "127.0.0.1";
```

Comment:
@vuong-buihv Only realized after the previous merge/PR, but won't IP always be localhost/127.0.0.1?
Therefore the user can't change via args (but PORT should definitely be modifiable 👍).